packages/react-composites/src/composites/CallComposite/components/LocalDeviceSettings.tsx

```diff
@@ -30,6 +30,7 @@ import { useSelector } from '../hooks/useSelector';
 import { getRole, getVideoEffectsDependency } from '../selectors/baseSelectors';
 import { getEnvironmentInfo } from '../selectors/baseSelectors';
 import { _isSafari } from '../utils';
+import { useId } from '@fluentui/react-hooks';
```

Comment:
Are we allowed to use this hook to set ID's? that would make some of our issues that we have easier to solve if we can

Reply:
We can't use react18s use id, but don't think there's an issue with fluent's?

Reply:
Yep should be fine https://github.com/microsoft/fluentui/blob/eb5c486ba272d82f8661da4d5deea5a1291ca5aa/packages/react-hooks/src/useId.ts#L12